packages/headless/src/features/search-and-folding/build-search-request.ts

```diff
@@ -82,6 +58,23 @@ function buildConstantQuery(state: StateNeededByExecuteSearchAndFolding) {
     (tab) => tab.isActive
   );
   const tabExpression = activeTab?.expression.trim() || '';
+  const filterExpressions = getStaticFilterExpressions(state);
```

Comment:
Since we are in the `search and folding` file, I think we currently have a bug regarding the "load all conversation" folding feature.

Generally speaking, this feature should not take into account any end user filter (facets, tabs, "static filter"), in order to really "load a whole conversation".


That's my bad, since I should have removed the `tabExpression` logic when fixing folding problem couple weeks ago.

I think we should keep `cq` only here (that's what I would assume to be filter set programmatically by code, and not by the end user), and in `search-actions` -> `buildSearchRequest()` function is where we should do the concatenation for `cq` + `tab` + `static filter`.